packages/run-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -10,8 +10,8 @@ import { toVaultKey } from './storeUtils.js';
 
 /**
  *
- * @param {Amount<NatValue>} debtAmount
- * @param {Amount<NatValue>} collateralAmount
+ * @param {Amount<'nat'>} debtAmount
+ * @param {Amount<'nat'>} collateralAmount
```

Comment:
Yes, `tsc` should grok a `const NAT = 'nat';` thing, I'm told (and I've seen).

Whether it's smart enough for `const { NAT } = AssetKind;`, I don't know.

Reply:
> concerned about the opportunity for typos

The type parameter K must be a subtype of `AssetKind` which is the union of `'nat' | 'set' | 'copySet' | 'copyBag'`, so if someone were to type any string other than one of those four it would be a type failure. ([type](https://github.com/Agoric/agoric-sdk/blob/7e1e93241a5cd88a5931240c66bddbdab86b939e/packages/ERTP/src/types.js#L55))

There may be an aesthetic argument to use `AssetKind.NAT` ([const](https://github.com/Agoric/agoric-sdk/blob/7e1e93241a5cd88a5931240c66bddbdab86b939e/packages/ERTP/src/amountMath.js#L14-L25)) but it would be more verbose without any safety benefit: `Amount<AssetKind.NAT>`.

Dan's `Amount<NAT>` idea would b no more verbose and a little cleaner but then you'd have to define or import NAT in the file you're typing that. (That import works, btw, because AssetKind object's type is a not `[index]: string` but literally each key to each value.)

Given how easy this would be to revert or change to either of the above, can we let it through and adjust as needed?

Reply:
Second thought, this PR is unblocking and the types are just nice to have so I extracted to https://github.com/Agoric/agoric-sdk/pull/4736

Reply:
> Given how easy this would be to revert or change to either of the above, can we let it through and adjust as needed?

I don't feel comfortable speaking for @erights on that.